### PR TITLE
[cli] Filter extra avd info when listing emulators

### PR DESCRIPTION
**Pull request by gabrieldonadel**
# Why

Recently got a few reports from @brentvatne and @Simek regarding a non bootable device being listed in Orbit
<details>
<summary>Check image</summary>

image

</details>

After doing some testing this same bug also applies to expo-cli 

```
yarn expo run:android
yarn run v1.22.21
$ /Users/brent/code/microfoam/node_modules/.bin/expo run:android
✔ Created native directory
✔ Updated package.json | no changes
✔ Finished prebuild
› Opening emulator INFO    | Storing crashdata in: /tmp/android-brent/emu-crash-34.1.18.db, detection is enabled for process: 92192
Error: The emulator (INFO    | Storing crashdata in: /tmp/android-brent/emu-crash-34.1.18.db, detection is enabled for process: 92192) quit before it finished opening. You can try starting the emulator manually from the terminal with: /Users/brent/Library/Android/sdk/emulator/emulator @INFO    | Storing crashdata in: /tmp/android-brent/emu-crash-34.1.18.db, detection is enabled for process: 92192
Error: The emulator (INFO    | Storing crashdata in: /tmp/android-brent/emu-crash-34.1.18.db, detection is enabled for process: 92192) quit before it finished opening. You can try starting the emulator manually from the terminal with: /Users/brent/Library/Android/sdk/emulator/emulator @INFO    | Storing crashdata in: /tmp/android-brent/emu-crash-34.1.18.db, detection is enabled for process: 92192
    at stopWaitingAndReject (/Users/brent/code/microfoam/node_modules/@expo/cli/build/src/start/platforms/android/emulator.js:110:20)
    at ChildProcess.<anonymous> (/Users/brent/code/microfoam/node_modules/@expo/cli/build/src/start/platforms/android/emulator.js:121:13)
    at ChildProcess.emit (node:events:517:28)
    at ChildProcess._handle.onexit (node:internal/child_process:292:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
# How

AVD IDs cannot contain spaces,  so in order to remove all the extra info returned by `emulator -list-avds`, we can just filter out lines that contain spaces 

# Test Plan

Running `yarn expo run:android -d` should not list ` INFO    | Storing crashdata in: /tmp/android-brent/emu-crash-34.1.18.db, detection is enabled for process: 92342 (emulator)` as an option to launch

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by byCedric** [comment]
Related: https://github.com/expo/expo/pull/27473

File: packages/@expo/cli/src/start/platforms/android/emulator.ts [hunk 1]
```diff
@@ -23,16 +23,23 @@ export function whichEmulator(): string {
...
-      }));
+    return (
+      stdout
+        .split(os.EOL)
+        .filter(Boolean)
+        /**
+         * AVD names cannot contain spaces. This removes extra info lines from the output. e.g.
+         * "INFO    | Storing crashdata in: /tmp/android-brent/emu-crash-34.1.18.db
+         */
+        .filter((name) => !name.trim().includes(' '))
```
Comment (byCedric) [inline packages/@expo/cli/src/start/platforms/android/emulator.ts:34]: I don't think this is true, [from Google's AVD docs](https://developer.android.com/studio/run/managing-avds#avdproperties)

image

**Review by byCedric — CHANGES_REQUESTED** [review]

File: packages/@expo/cli/src/start/platforms/android/emulator.ts (hunk 1 above)
Comment (gabrieldonadel) [reply packages/@expo/cli/src/start/platforms/android/emulator.ts:34]: oh, I didn't mean that type of "name", I guess the correct term would be "AVD ID". Even if you rename an AVD through Android studio the returned id when running `emulator -list-avds`  will never include spaces. let me update the comment

Comment (gabrieldonadel) [reply packages/@expo/cli/src/start/platforms/android/emulator.ts]: updated the comment

Comment (byCedric) [reply packages/@expo/cli/src/start/platforms/android/emulator.ts]: Gotcha, can confirm:

image

**Review by byCedric — APPROVED** [review]

**Pull request merged**